packages/spec-api/src/types.ts

```diff
@@ -50,6 +50,23 @@ export interface MockApi {
   handler: MockRequestHandler;
 }
 
+export interface MockApiDefinition {
+  uri: string;
+  mockMethods: MockMethod[];
+}
+
+export interface MockMethod {
+  method: HttpMethod;
+  request: ServiceRequest;
+  response: any;
+  handler?: MockRequestHandler;
+}
+
+export interface ServiceRequest {
+  body?: any;
+  config?: any | undefined;
```

Comment:
Please make this not be tied to axios config.

Reply:
@timotheeguerin  I agree that the original idea of config was taken from axios. But, it makes sense to keep all config related properties (`headers`, `params`, etc) under one object instead of multiple properties. Don't you think?

Reply:
i disagree, there is nothing that sounds like config that groups params and headers but why not body. 
coming form never have used axios before this just sounds like an aribtrary grouping.

I think if anything is should closer to the `RequesetInit` type used by the standard fetch.

Reply:
Done.